util/convertnhcb/convertnhcb.go

```diff
@@ -139,21 +139,17 @@ func (h TempHistogram) Convert() (*histogram.Histogram, *histogram.FloatHistogra
 		return nil, nil, h.err
 	}
 
+	if !h.hasCount && len(h.buckets) > 0 {
+		// No count, so set count to the highest known bucket's count.
+		h.count = h.buckets[len(h.buckets)-1].count
+	}
```

Comment:
Should `h.hasCount` be set to `true` here? This was previously done on L154 so wondering if it's necessary here as well.

Reply:
I think it matters less here as hasCount is needed for Convert method which is happening here. But as a good practice a comment explaining this or settting true might be more clear yes 👍🏽

Reply:
I'll add setting hasCount, that's cleanest and expected, thanks!